src/components/InstructionButton.js

```diff
@@ -9,19 +9,22 @@ import Glyphicon from 'react-bootstrap/lib/Glyphicon';
 import {getTranslator} from '../selectors/translate';
 import {getLessonPath} from '../resources/lessonFrontmatter';
 
-const InstructionButton = ({course, lesson, language, isReadme, onlyIcon, insideLink, buttonText}) => {
-  const buttonArgs = {
+const InstructionButton = ({course, lesson, language, isReadme, onlyIcon, insideLink, buttonText}, {router}) => {
+  const path = getLessonPath(course, lesson, language, isReadme);
+  const options = {
     className: onlyIcon ? styles.buttonOnlyIcon : styles.button,
     bsStyle: 'guide',
     bsSize: onlyIcon ? 'xs' : 'small',
     componentClass: insideLink ? 'div' : 'a',
+    tabIndex: '0',
+    'aria-label': buttonText,
+    onKeyPress: () => router.push(path),
```

Comment:
Hm, her har vi en arrow-function inni render... Burde vi gjøre InstructionButton om til en PureComponent, og flytte onKeyPress-definisjonen utenfor render?

Reply:
Ser at det er flere steder src det er komponenter med arrow-functions. Kan jeg lage et issue på dette og la denne stå for nå?